deltachat-ios/Controller/ClosedAccountErrorViewController.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  ClosedAccountErrorViewController.swift
+//  deltachat-ios
+//
+//  Created by bb on 05.07.22.
+//  Copyright © 2022 Jonas Reinsch. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import DcCore
+
+class ClosedAccountErrorView: UIView {
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Error: Account is closed"
+        label.textColor = DcColors.grayTextColor
+        label.textAlignment = .center
+        label.numberOfLines = 1
+        label.font = UIFont.systemFont(ofSize: 24, weight: .bold)
+        return label
+    }()
+    
+    private lazy var errorDescription1: UILabel = {
+        let label = UILabel()
+        label.text = "This should not happen, please report it to the developers"
+        label.textColor = DcColors.grayTextColor
+        label.textAlignment = .center
+        label.numberOfLines = 1
+        label.font = UIFont.systemFont(ofSize: 20, weight: .bold)
+        return label
+    }()
+    
+    private lazy var errorDescription2: UILabel = {
+        let label = UILabel()
+        label.text = "Anyways: just kill and restart the app this should fix the error."
+        label.textColor = DcColors.grayTextColor
+        label.textAlignment = .center
+        label.numberOfLines = 1
+        label.font = UIFont.systemFont(ofSize: 20, weight: .bold)
+        return label
+    }()
+    
+    
+    public init () {
+        super.init(frame: .infinite)
+        
+        addSubview(titleLabel)
+        addSubview(errorDescription1)
+        addSubview(errorDescription2)
+        
```

Comment:
here you need to add your constraints - how do you want to arrange your UIElements?